doc/ref_impl/cntt-ri2/chapters/chapter03.md

```diff
@@ -16,14 +16,14 @@
 
 The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves two purposes which guide the definition of requirements in this chapter:
 
-1. RI2 serves as a baseline and proving ground for developing the [Reference Conformance 2 (RC2) specification](../../ref_cert/RC2) as well as the Cloud Native OVP program (LINK), and
+1. RI2 serves as a baseline and proving ground for developing the [Reference Conformance 2 (RC2) specification](../../../ref_cert/RC2) as well as the [Cloud Native OVP](https://wiki.lfnetworking.org/x/agCDAQ) program.
```

Comment:
RI2 is verified by RC2 to onboard CNF.
RC2 shouldn't be developed on RI2 (it's the control loop

Reply:
@collivier could you suggest some language that makes it clearer? (For me, the current wording just means we verify RI2 using RC2 as a means to validating that RC2 is delivering what is needed, not that RI2 is the development platform.)

Reply:
@collivier @tomkivlin Should we create a ticket specifically for the above? This PR has been lingering for almost a month, so it would be good to get it merged

Reply:
I've created #2125 to track the updated purpose related to RC2